styles.css

```diff
@@ -538,4 +528,264 @@ textarea::placeholder {
   .see-project-btn {
     width: 50%;
   }
+
+  .js-img img,
+  .html-img img,
+  .css-img img {
+    margin-top: 1rem;
+    width: 90%;
+    border-radius: 0.5rem;
+  }
+
+  .framework-list,
+  .skill-list {
+    display: flex;
+    justify-content: space-between;
+    padding: 1rem;
+  }
+}
+
+@media only screen and (min-width: 768px) {
+  .header {
+    padding-left: 155px;
+  }
+
+  .nav {
+    margin-right: 154px;
```

Comment:
- [ ] Good work on your headline section but there are some aspects you can improve on
     -  Kindly increase the width of your headline section so that it covers a larger area like in the original design especially the paragraph
     - Kindly use the white background color,`ffffff` used in the original design. This will make the border radius on the bottom left of the page to be visible. 
     
![image](https://user-images.githubusercontent.com/98453979/220683043-801f6a3c-72eb-414c-82c5-44bc9358c8db.png)

### Checkout the original design

![image](https://user-images.githubusercontent.com/98453979/220683575-09ff11bd-589f-4c77-98fa-2189db790733.png)